politeiawww/cmd/politeiawwwcli/client/client.go

```diff
@@ -931,6 +931,32 @@ func (c *Client) UserDetails(userID string) (*v1.UserDetailsReply, error) {
 	return &udr, nil
 }
 
+func (c *Client) Users(email, username string) (*v1.UsersReply, error) {
+	u := v1.Users{
```

Comment:
Small nitpick.  Can you move this struct initialization over to the the `UsersCmd.Execute` function and then pass it in as a argument to `Users()`?  This would make it consistent with how the rest of the client does it.  This is something that was changed in order to get more control over the stdout logging.